scripts/glibc_check.py

```diff
@@ -35,12 +35,12 @@ def main():
             output = subprocess.check_output(objdump % filename, shell=True,
                                              stderr=subprocess.STDOUT)
             output = output.decode()
-            versions = set(parse_version(match.group(1))
-                           for match in glibc_re.finditer(output))
+            versions = {parse_version(match.group(1))
+                           for match in glibc_re.finditer(output)}
```

Comment:
hmm, do we really want to *fix* such stuff in pre-commit, trusting that it always is correct and acceptig that it does not fix the indentation?

Reply:
Typically it's used in conjunction with the flake8 and the black hook, but that's way too much at once

I don't consider it always right, but it is a massive help for finger memory sediments